self.device  cpu
/home/lcur1101/ATCS_group3/marc_data/dataset_fr_train.json
init marc done
num_lines  200000
num_lines  200000
num_lines  200000
num_lines  200000
Batch number: 0 , batch size : 32
Time taken to execute prompt gen: 0:00:00.000056
Time taken to execute mapping: 0:00:00.000236
Traceback (most recent call last):
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 174, in <module>
    # parser.add_argument('--datasetpath', type=str, help='Path to the dataset')
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 116, in pipeline
    answers_probs_batch, pred_answer_batch = model(
  File "/home/lcur1101/ATCS_group3/code/main_model.py", line 45, in __call__
    answers_probs[:, idx] = probs#.T
RuntimeError: expand(torch.FloatTensor{[32, 3]}, size=[32]): the number of sizes provided (1) must be greater or equal to the number of dimensions in the tensor (2)
srun: error: r28n5: task 0: Exited with exit code 1
